packages/env/README.md

```diff
@@ -365,3 +375,4 @@ You can tell `wp-env` to use a custom port number so that your instance does not
 ```
 
 <br/><br/><p align="center"><img src="https://s.w.org/style/images/codeispoetry.png?1" alt="Code is Poetry." /></p>
+````
```

Comment:
Wasn't expecting to see this added. Maybe an artefact from a previous merge conflict?

```suggestion
```

Reply:
Yeah, that's weird!